opslevel/resource_opslevel_service.go

```diff
@@ -184,7 +192,7 @@ func resourceServiceCreate(d *schema.ResourceData, client *opslevel.Client) erro
 		Language:    d.Get("language").(string),
 		Framework:   d.Get("framework").(string),
 		Tier:        d.Get("tier_alias").(string),
-		Owner:       d.Get("owner_alias").(string),
+		Owner:       opslevel.NewIdentifier(d.Get("owner").(string)),
```

Comment:
we cannot do this this.  The customer could still be using `owner_alias` so we need to reconcile both of those fields.  

- If both are set we should throw an error
- if `owner_alias` is set use that
- if `owner` is set use that

Reply:
Re-added `owner_alias` in resource